tests/cupy_tests/linalg_tests/test_norms.py

```diff
@@ -157,11 +163,18 @@ def test_slogdet_4(self, xp, dtype):
 
     @testing.for_float_dtypes(no_float16=True)
     @testing.numpy_cupy_allclose(rtol=1e-3, atol=1e-4)
-    def test_slogdet_fail(self, xp, dtype):
+    def test_slogdet_singular(self, xp, dtype):
+        a = xp.zeros((3, 3), dtype)
+        sign, logdet = xp.linalg.slogdet(a)
+        return xp.array([sign, logdet], dtype)
```

Comment:
This ways is combining multiple arrays is not very good, but this PR is not responsible of that.